benchmarks/torchbench.py

```diff
@@ -59,7 +59,6 @@
     "detectron2_fasterrcnn_r_50_fpn",
     "detectron2_maskrcnn_r_101_c4",
     "detectron2_maskrcnn_r_101_fpn",
-    "detectron2_maskrcnn_r_50_c4",
```

Comment:
Why dropping this one?

Reply:
It's the other way around. I am actually removing this from the skip list, so that it shows up in nightly.